UI/src/less/site/asset-form.less

```diff
@@ -365,7 +365,14 @@
     text-transform: uppercase;
   }
 
-  .wrong-speed-limits {
+  .wrong-speed-limits{
+    position:absolute;
+    right: 10px;
+    text-transform: uppercase;
+    bottom: 128px;
+  }
+
+  .wrong-linear-assets{

```

Comment:
this could be change, since the rules are the same; 

` .wrong-speed-limits .wrong-linear-assets{
    position:absolute;
    right: 10px;
    text-transform: uppercase;
    bottom: 128px;
  }`